src/pages/users/index.tsx

```diff
@@ -119,23 +118,24 @@ const UserRow = ({ user, index }: { user: User; index: number }) => {
     >
       <th
         scope="row"
-        className="whitespace-nowrap px-6 py-4 font-medium text-gray-900 "
+        className="whitespace-nowrap px-6 py-4 font-medium text-gray-900"
       >
         {user.firstName}
       </th>
-      <td className="px-6 py-4">{user.lastName}</td>
+      <td className="px-6 py-4 capitalize">{user.lastName}</td>
```

Comment:
why is last name not capitalised but first name is?

Reply:
What if the user has my last name...